website/source/docs/providers/archive/r/file.html.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: "archive"
+page_title: "Archive: archive_file"
+sidebar_current: "docs-archive-resource-file"
+description: |-
+  Generates an archive from content, a file, or directory of files.
+---
+
+# archive\_file
+
+Generates an archive from content, a file, or directory of files.
+
+## Example Usage
+
+```
+resource "archive_file" "init" {
+    template = "${file("${path.module}/init.tpl")}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+NOTE: One of `source_content_filename` (with `source_content`), `source_file`, or `source_dir` must be specified.
+
+* `type` - (required) The type of archive to generate.
+  NOTE: `archive` is supported.
```

Comment:
Good find!  I guess I was a little aggressive when I renamed `zip` to `archive`.